source/WindowScroller/WindowScroller.css

```diff
@@ -0,0 +1,3 @@
+.disabled > * {
+  pointer-events: none !important;
```

Comment:
`pointer-evetns` is an Inherited style, yes, but it can be overridden by children. And `Grid` would override, since it sets the property on the `ReactVirtualized__Grid__innerScrollContainer`, which is not the direct child of `WindowScroller` but- at best- a child of its child.

Also using `!important` is kind of a last resort, so I like to avoid it.